includes/discovery/os/nos.inc.php

```diff
@@ -4,4 +4,7 @@
     if (strstr($sysDescr, "Brocade VDX")) {
```

Comment:
You could just do 
if (strstr($sysDescr, "Brocade VDX") II strstr($sysDescr, "BR-VDX")) {